application/controllers/rabbitmq_submitter.py

```diff
@@ -1,6 +1,6 @@
 from pika import BlockingConnection, BasicProperties, URLParameters
 from pika.exceptions import AMQPError
-from ras_common_utils.ras_error.ras_error import RasError
+from application.exceptions import RasError
```

Comment:
Don't think this adheres to PEP8 import standards. If you are importing from the application itself this should be a third group of imports at the end of the rest of imports.